zerver/lib/actions.py

```diff
@@ -1964,7 +1964,9 @@ def do_send_messages(
                 online_push_enabled=(user_id in send_request.online_push_user_ids),
                 stream_push_enabled=(user_id in send_request.stream_push_user_ids),
                 stream_email_enabled=(user_id in send_request.stream_email_user_ids),
-                wildcard_mention_notify=(user_id in send_request.wildcard_mention_user_ids),
+                wildcard_mentions_notifications_enabled=(
+                    user_id in send_request.wildcard_mention_user_ids
+                ),
```

Comment:
This one is not pure configuration; see this part of `send_request` construction:
```
    if message_dict["message"].mentions_wildcard:                                          
        wildcard_mention_user_ids = info["wildcard_mention_user_ids"]                      
    else:                                                                                  
        wildcard_mention_user_ids = set()                                                  
```

So we want a name that encodes "wildcard mentions and notifications are enabled".

Reply:
Yeah, my bad. How about `wildcard_mention_notifiable_user_ids`? I think that makes the most sense.